### DISCUSSION TITLE
JS and Core Can Use a cluster together

### DISCUSSION THREAD

**Discussion by ltDamon** [nats-io/nats-server#2961] [General]
If I want to use these two features at the same time，JetStream and Core. I must to deploy two cluster?

![image](https://user-images.githubusercontent.com/62693485/160092823-c516e61f-ebe4-423a-9527-8d9c7a8a852f.png)

**Comment by wallyqs** [comment]
You can use both at the same time with the same cluster.

**Comment by ltDamon** [comment]
It said  where none of the nats-server instances are configured to enable JetStream. Just wanna check if i enable the JetStream. it affect the affect efficiency of "core nats".

**Comment by ripienaar** [comment]
Enabling jetstream will increase memory use on the server - and using it again also will increase memory use. 

But core nats performance will remain the same essentially
